ambari-server/src/main/resources/custom_action_definitions/system_action_definitions.xml

```diff
@@ -74,6 +74,17 @@
     <targetType>ALL</targetType>
     <permissions>HOST.ADD_DELETE_COMPONENTS, HOST.ADD_DELETE_HOSTS, SERVICE.ADD_DELETE_SERVICES</permissions>
   </actionDefinition>
+  <actionDefinition>
+    <actionName>mpack_packages</actionName>
```

Comment:
Should we remove the `install_packages` ?

Reply:
Surely.

Reply:
Done